src/util-inl.h

```diff
@@ -234,27 +234,39 @@ bool StringEqualNoCaseN(const char* a, const char* b, size_t length) {
 // compiler version specific functionality.
 // malloc(0) and realloc(ptr, 0) have implementation-defined behavior in
 // that the standard allows them to either return a unique pointer or a
-// nullptr for zero-sized allocation requests.  Normalize by always using
-// a nullptr.
+// nullptr for zero-sized allocation requests. Normalize by always using
+// a static buffer.
+static void* fake_mem = malloc(1);
```

Comment:
Are you sure this works as it is? Having `fake_mem` as a `static` variable means that every object file is going to end up with its own instance of that, meaning that e.g. the `Free()` from `node_crypto.cc` can’t handle `Malloc()`ed buffers from `node_buffer.cc`.